streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/Functions.scala

```diff
@@ -89,13 +89,57 @@ trait Functions extends SQlBaseFunc with Logging with WowLog with Serializable {
       if (params.contains(f.name)) {
         val v = params(f.name)
         val m = model.getClass.getMethods.filter(m => m.getName == s"set${f.name.capitalize}").head
+        val pa = m.getParameters.head.toString.toLowerCase()
```

Comment:
What's pa?  do you want to support the parameter type is array or complex object? If so, this seems not a good idea.

As we know, we should set the model value dynamically from where statement the condition is a string,  but the model value is a complex object, the best way is to add more case statement to `configureModel` method but not change the exists case statement.

Or you can just write you own configureModel in your subclass if it's not a common case.